From 42512bbf83b74e12bfb05f619645a99528680d0a Mon Sep 17 00:00:00 2001
From: Apollon Oikonomopoulos <apoikos@gmail.com>
Date: Mon, 21 Nov 2016 12:21:19 +0200
Subject: [PATCH] Drop dependency on MonadCatchIO-transformers

MonadCatchIO-transformers is deprecated and has been removed from Debian
unstable. Replace it with Control.Exception.
---
 cabal/ganeti.template.cabal   | 1 -
 src/Ganeti/Metad/WebServer.hs | 6 +++---
 2 files changed, 3 insertions(+), 4 deletions(-)

--- a/cabal/ganeti.template.cabal
+++ b/cabal/ganeti.template.cabal
@@ -65,7 +65,6 @@
     , lens                          >= 3.10
     , lifted-base                   >= 0.2.0.3    && < 0.3
     , monad-control                 >= 0.3.1.3    && < 1.1
-    , MonadCatchIO-transformers     >= 0.3.0.0    && < 0.4
     , network                       >= 2.3.0.13   && < 2.7
     , parallel                      >= 3.2.0.2    && < 3.3
     , regex-pcre                    >= 0.94.2     && < 0.95
